specs/commandline/plugin.md

```diff
@@ -36,15 +36,55 @@ Aliases:
   list, ls
 ```
 
+### notation plugin install
+
+```text
+Installs a plugin
+
+Usage:
+  notation plugin install [--name <plugin name>] [flags] <plugin package> 
+
+Flags:
+  -n, --name           string   name of the plugin
+  -p, --plugin-package string   path to the plugin package
+  -h, --help                    help for install
+
+Aliases:
+  install, add
+```
+
+### notation plugin remove 
+
+```text
+Removes a plugin
+
+Usage:
+  notation plugin remove [flags] <plugin name>
+
+Flags:
+  -h, --help          help for remove
+
+Aliases:
+  remove, rm, uninstall, delete
+```
+
 ## Usage
 
 ### Install a plugin
 
-Currently there is no subcommand available for plugin installation. Plugin publisher should provide instructions to download and install the plugin.
+```shell
+notation plugin install --name <plugin name> --plugin-package <plugin package>
```

Comment:
If the `--name` option does not present, we need to prompt the user if the plugin is intended.

Reply:
Resolving per request of @yizha1, please add additional comments if you disagree or have questions with my previous response.

Reply:
Instead of prompting should we just emit the plugin name ?

Reply:
I agree with @priteshbandi. Having the user type `--name pluginName` validate the plugin adds enough friction that it's a deterrent which reduces the chances that it'll be used for validation. 

Emitting the plugin name might be a better option until it adds more value. For example, if the plugin-name provided a list of trusted plugins you could select from and then the CLI tool downloads and installs it for you.

Reply:
I'll update the spec to emit the `--name` option.